universe/storage/s3/wire.go

```diff
@@ -97,32 +97,29 @@ func createClient(ctx context.Context, factory client.ClientFactory, minioCreds
 
 	loadOptFns = append(loadOptFns, config.WithRegion(region))
 
-	var cfg aws.Config
-	var err error
-	if *minioEndpoint != "" {
-		core.Log.Printf("[storage/s3] creating minio client: %s", *minioEndpoint)
-
-		resolver := aws.EndpointResolverWithOptionsFunc(func(service, region string, options ...interface{}) (aws.Endpoint, error) {
-			return aws.Endpoint{
-				PartitionID:   "aws",
-				URL:           *minioEndpoint,
-				SigningRegion: region,
-			}, nil
-		})
+	if *minioEndpoint == "" {
+		return nil, fmt.Errorf("aws S3 is not supported")
```

Comment:
Aren't you removing the irsa path here as well? I.e. when `AWS_WEB_IDENTITY_TOKEN_FILE` is set. This will break internal, where we use s3.

Reply:
Reverted, I initially started to delete more than needed but then forgot to revert this path.